Manual.md

```diff
@@ -146,6 +146,10 @@ fonts).
 Browser forks, including those based on Chromium and Firefox, are generally not
 accepted. Such forks require heavy patching, maintenance and hours of build time.
 
+Package need to be provided in version announced by authors as ready to use by
+general public. Betas, arbitrary VCS revisions, templates using tip of development
+branch taken at build time and releases done by package maintainer won't be accepted.
```

Comment:
```suggestion
Packages should use versions announced by authors as ready to use by the
general public - usually called releases. Betas, arbitrary VCS revisions, templates using tip of development
branch taken at build time and releases done by package maintainer won't be accepted.
```

I would rather list "Alphas" than "Betas", because some upstreams tag a stable release every decade, but release "betas" in pretty good cadence, which are what everyone uses. We do have some exceptions in place, but idk how to document that.

Reply:
> rather list "Alphas" than "Betas"

I conciously made muddy what we accept, because of usage of pre-releases (wine) or arbitrary VCS revisions (nawk, iana-etc).

> should

I also prefer stronger wording here and handling some things when needed as _exceptions_ - rarely, when needed.

Reply:
> I conciously made muddy what we accept, because of usage of pre-releases (wine) or arbitrary VCS revisions (nawk, iana-etc).

Also see mtpaint. The author does not tag releases - announces them in a commit message.

Reply:
@Chocimier updated the suggestion, then.

Reply:
Thanks, updated.